tests/std/tests/P0220R1_any/test.cpp

```diff
@@ -3160,6 +3184,27 @@ namespace msvc {
         }
 #pragma warning(pop)
     } // namespace trivial
+
+    namespace gh_140_robust_against_adl {
+        struct incomplete;
+
+        template <class T>
+        struct wrapper {
+            T t;
+        };
+
+        void run_test() {
+            using _trivial = wrapper<incomplete>*;
+            using _small   = std::pair<_trivial, small>;
+            using _large   = std::pair<_trivial, large>;
+
+            globalMemCounter.disable_allocations = true;
+            any a{_trivial{}};
+            any b{_small{}};
+            globalMemCounter.disable_allocations = false;

```

Comment:
This is a test-compile test; I'm adding `globalMemCounter.disable_allocations = true / false;` to make it able to pass the runtime check.